recipes/watset/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "2.8.2" %}
+
+package:
+  name: watset
+  version: {{ version }}
+
+source:
+  url: https://github.com/nlpub/watset-java/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 7a957f55f219fbda7c8102b27bb7d8037a0a61a70d9f63b90d7b4746fd35a64f
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - openjdk >=11
+    - maven
+  run:
+    - openjdk >=11
+
+test:
+  commands:
+    - watset --version
+
+about:
+  home: https://github.com/nlpub/watset-java
+  summary: 'An implementation of the Watset clustering algorithm for Java'
+  description: |
+    This is an open source implementation of the Watset algorithm for fuzzy
+    graph clustering (aka soft clustering). This implementation is
+    multi-threaded, extensible, contains several unit tests, and can be
+    easily embedded into any Java-based graph processing pipeline.
+    Also, it offers a convenient command-line tool for running Watset
+    over the edge lists in text files.
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: LICENSE
```

Comment:
You should also include a third party license report that you can generate with maven. See https://github.com/conda-forge/jive-feedstock/blob/85dab7d3b704d757ac740ff5826bae37ceb10c85/recipe/build.sh#L6 as example.

Reply:
Thank you, fixed!